### PR TITLE
Specs to use Model.fetch_all.

### PR DESCRIPTION
**Pull request by huba**
Just some specs.

### CODE REVIEW

File: spec/relaxo/model/document_spec.rb [hunk 1]
```diff
@@ -96,9 +106,22 @@ class Invoice::Transaction
...
 		database.commit(message: "Adding test model") do |dataset|
 			transaction.date = Date.today - 1
 			transaction.save(dataset)
 		end
+		
+		transactions = Invoice::Transaction.all(database.current)
+	end
+	
+	it "can query by index" do
+		database.commit(message: 'doot') do |changes|
```
Comment (ioquatix) [inline spec/relaxo/model/document_spec.rb]: what is doot?

Comment (ioquatix) [reply spec/relaxo/model/document_spec.rb]: what about "Add test users" or something :)

Comment (huba) [reply spec/relaxo/model/document_spec.rb]: Oops. That slipped my mind. 💀Doot is [spooky](https://youtu.be/hzPpWInAiOg).

**Comment by ioquatix** [comment]
awesome thanks :)

**Pull request merged**